### ISSUE TITLE
The recent hWheel support breaks AbsoluteMouse

### ISSUE THREAD

**Issue by algernon** [keyboardio/KeyboardioHID#15]
I noticed that with the recent change to support horizontal wheel, `AbsoluteMouse` broke, and stopped working on my Linux, with the following messages in dmesg:

When plugging the keyboard in:

```
[Wed Nov 29 09:59:17 2017] hid-generic 0003:1209:2301.00FE: ignoring exceeding usage max
```

Whenever pressing a mouse warp key:

```
[Wed Nov 29 09:59:21 2017] usb 3-14.4.2: input irq status -75 received
```

With the `hWheel` support backed out of `KeyboardioHID`, warp keys work again, and I don't get the initial warning from `hid-generic` either.

**Comment by obra** [comment]
can you commit the backout for now and pull arduinoboards up?

**Comment by algernon** [comment]
I could, but I don't have permissions on this repo.

**Comment by algernon** [comment]
GOOD NEWS!

I found the underlying issue, and there's an easy fix that allows us to have BOTH horizontal wheel AND warp!

In 78caf1ee3048dda7b10ae8f44011a428a9242a06, `AbsoluteMouse` was also changed to have horizontal wheel support, and THAT is too much for the OS to handle. As we don't use the wheels on `AbsoluteMouse` *at all*, it is safe to back that part of the change out, and leave `Mouse` with horizontal wheel support in.

I'll prep a PR in a bit that does just this.
